src/adapters/givPowerBalanceAggregator/givPowerBalanceAggregatorInterface.ts

```diff
@@ -0,0 +1,48 @@
+export interface GetLeastIndexedBlockTimeStampInputParams {
```

Comment:
```suggestion
export interface BANetworksParams {
```

Reply:
I would add BA prefix to all params
Also for this one, the name must be more general as other parameter types are extending it